src/pages/partners/partners.module.scss

```diff
@@ -0,0 +1,98 @@
+.colored_block {
+    // Not used yet
+    color: white;
+    background-color: $as-violet-1;
+    padding-block: 1.5em;
+    text-align: center;
+  }
+
+.container {
+  display: flex;
+  justify-content: center;
+}
+
+.introduction {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  max-width: 962px;
+  gap: 30px;
+}
+
+.intro_quote {
+  line-height: 25px;
+  font-family: Graphik;
```

Comment:
Let's use only the fonts imported in the main.css or basic system fonts. I suggest to either change this to serif or we could decide to have a global serif font for quotes.